recipes-openxt/acms/acms.bb

```diff
@@ -43,18 +43,21 @@ LIC_FILES_CHKSUM = "file://GM45_GS45_PM45-SINIT_51/license.txt;md5=60d123634e0b9
                     file://7th_gen_i5_i7-SINIT_74/license.txt;md5=4897766beede80773ddd162b2724a8bb \
 "
 
-SRC_URI = "${OPENXT_MIRROR}/GM45_GS45_PM45-SINIT_51.zip;name=gm45 \
-           ${OPENXT_MIRROR}/Q45_Q43-SINIT_51.zip;name=q45 \
-           ${OPENXT_MIRROR}/Q35-SINIT_51.zip;name=q35 \
-           ${OPENXT_MIRROR}/i5_i7_DUAL-SINIT_51.zip;name=i5 \
-           ${OPENXT_MIRROR}/i7_QUAD-SINIT_51.zip;name=i7 \
-           ${OPENXT_MIRROR}/Xeon-5600-3500-SINIT-v1.1.zip;name=xeon_5600 \
-           ${OPENXT_MIRROR}/Xeon-E7-8800-4800-2800-SINIT-v1.1.zip;name=xeon_e7 \
-           ${OPENXT_MIRROR}/3rd-gen-i5-i7-sinit-67.zip;name=ivb_snb \
-           ${OPENXT_MIRROR}/4th-gen-i5-i7-sinit-75.zip;name=hsw \
-           ${OPENXT_MIRROR}/5th_gen_i5_i7-SINIT_79.zip;name=bdw \
-           ${OPENXT_MIRROR}/6th_gen_i5_i7-SINIT_71.zip;name=skl \
-           ${OPENXT_MIRROR}/7th_gen_i5_i7-SINIT_74.zip;name=kbl \
+MIRRORS_append += " \
+    http://.*/.*    https://openxt.ainfosec.com/mirror/ \
+"
+SRC_URI = "${OPENXT_MIRROR}/openxt/GM45_GS45_PM45-SINIT_51.zip;name=gm45 \
```

Comment:
Should "/openxt" just be added to the OPENXT_MIRROR definition?  I don't know how MIRRORS_append works, but https://openxt.ainfosec.com/mirror/openxt/GM45_GS45_PM45-SINIT_51.zip does not exist - it does without the extra openxt directory.

Reply:
@jandryuk [`MIRRORS`](https://www.yoctoproject.org/docs/2.4.2/ref-manual/ref-manual.html#var-PREMIRRORS) will replace matching `http://.*/.*` (so expanding `OPENXT_MIRROR` -> `http://mirror.openxt.org/openxt`) with `https://openxt.ainfosec.com/mirror` (at least that is the behavior I have seen so far) and fetch, for example, https://openxt.ainfosec.com/mirror/GM45_GS45_PM45-SINIT_51.zip. We could add `/openxt` to `OPENXT_MIRROR` though. I believe the confusion came because some acms where initially (presumed?) to live in http://mirror.openxt.org/acms.